src/DynamoCoreWpf/Extensions/ViewStartupParams.cs

```diff
@@ -27,12 +27,20 @@ public IExtensionManager ExtensionManager
         internal ViewStartupParams(DynamoViewModel dynamoVM) :
             base(dynamoVM.Model.AuthenticationManager.AuthProvider,
                 dynamoVM.Model.PathManager,
-                new ExtensionLibraryLoader(dynamoVM.Model), 
-				dynamoVM.Model.CustomNodeManager,
+                new ExtensionLibraryLoader(dynamoVM.Model),
+                dynamoVM.Model.CustomNodeManager,
                 dynamoVM.Model.GetType().Assembly.GetName().Version,
                 dynamoVM.Model.PreferenceSettings)
         {
             dynamoViewModel = dynamoVM;
         }
+
+        public DynamoViewModel DynamoViewModel
+        {
+            get
+            {
+                return dynamoViewModel;
```

Comment:
I know, that @pboyer , won't like this change. But we can't create Mutator  without `DynamoViewModel`.